src/utils.rs

```diff
@@ -50,3 +59,33 @@ pub fn parse_gj_messages_response(meow: String) -> Result<Vec<GDMessage>, Generi
         })
         .collect()
 }
+
+pub fn valid_url_check(url: &str) -> bool {
+    let reg = regex::Regex::new(r#"(?i)^(http://www\.|https://www\.|http://|https://)?[a-z0-9]+([\-\.]{1}[a-z0-9]+)*\.[a-z]{2,5}(:[0-9]{1,5})?(/.*)?$"#).unwrap();
+
+    if reg.is_match(&url) {
+        true
+    } else {
+        println!("{} {} ({})", Red.bold().paint("Invalid URL set in"), Yellow.bold().paint("\"servers.json\""), url);
+        false
+    }
+}
+
+pub async fn proxy_list() -> Result<String, GenericError> {
+    let srv: String = if std::fs::File::open("servers.json").is_err() {
+        BOOMLINGS_SERVER.to_string()
+    } else {
+        let file = std::fs::read_to_string("servers.json")?;
+        let s: Servers = serde_json::from_str(&file).unwrap();
```

Comment:
json syntax errors will make this panic due to .unwrap usage without a graceful way to handle the error

Reply:
fixed in 00c151a